tstypes/type/definition.d.ts

```diff
@@ -387,6 +387,7 @@ export class GraphQLObjectType<
   toConfig(): GraphQLObjectTypeConfig<any, any> & {
     interfaces: GraphQLInterfaceType[];
     fields: GraphQLFieldConfigMap<any, any>;
+    extensions: Maybe<Readonly<Record<string, any>>>;
```

Comment:
@mike-marcacci Thanks for fixing I went ahead so and fixed all similar stuff in #2202 sorry for rebases but it will keep cleaner diff in this PR.

Reply:
No worries – it's easy as long as I don't let it get too far out of date :)